README.rst

```diff
@@ -1,37 +1,44 @@
-==========
-``ci_hsc_gen3``
-==========
+######
+ci_hsc
+######
 
-``gen2_ci_hsc`` provides scripts which use the LSST stack to perform single frame and coadd processing based
-on engineering test data from Hyper Suprime-Cam.
+``ci_hsc`` provides scripts which use the LSST stack to perform single frame and coadd processing based on engineering test data from Hyper Suprime-Cam.
 
 Obtaining test data
 ===================
 
-The data used by ``gen2_ci_hsc`` is linked against data installed by ``testdata_ci_hsc``, please
-setup that package before running scons on this one.
+``ci_hsc`` uses the the files in `testdata_ci_hsc`_ to run its tests.
+That package must be setup first, before running ``scons`` (see below).
+
+.. _testdata_ci_hsc: https://github.com/lsst/testdata_ci_hsc/
 
 Running the tests
 =================
 
 Set up the package
 ------------------
 
-The package must be set up in the usual way before running::
+You must first setup `testdata_ci_hsc`_, before setting up this package with the ``-k`` option, to keep the test data package set up::
 
-  $ cd ci_hsc_gen3
-  $ setup -j -r .
+  $ cd PATH_TO_TESTDATA_CI_HSC
+  $ setup -r .
+  $ cd PATH_TO_CI_HSC
+  $ setup -kr .
 
 Running the tests
 -----------------
 
-Execute ``scons``. On a Mac running OSX 10.11 or greater, you must specify a
-Python interpreter followed by a full path to ``scons``::
+Note that running these tests can take a few hours hours, depending on the speed of your machine and the number of cores available.
+Execute ``scons -jN``, where ``N`` is the number of CPU cores to use.
+On a Mac running OSX 10.11 or greater, you must specify a Python interpreter followed by a full path to ``scons``::
```

Comment:
I don't think that's a problem when scons comes from conda because the shebang is using the full path to the conda python binary and not `/usr/bin/env python`. I think this was true when scons was installed by eups.

Reply:
Ah, I wasn't sure about that line, and don't have a macOS machine to test it on. I suspect you're correct though; I'll remove it.

Reply:
I just typed `scons` on my mac and ci_hsc_gen3 started without complaint.